src/main/java/gil/server/JSONDataStore.java

```diff
@@ -0,0 +1,106 @@
+package gil.server;
+
+import javax.json.Json;
+import javax.json.JsonArray;
+import javax.json.JsonObject;
+import javax.json.JsonArrayBuilder;
+import javax.json.JsonObjectBuilder;
+import javax.json.JsonReader;
+import javax.json.stream.JsonParsingException;
+import java.io.*;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import static java.util.stream.Collectors.toList;
+
+
+public class JSONDataStore {
+    private List<Person> people = new ArrayList<>();
```

Comment:
Correct me if I'm wrong but isn't it best practice to state what kind of an arraylist you want to create? So this should probably read `new ArrayList<Person>()` no?

Reply:
Also, looking at this file, it doesn't seem to live upto its name. It seems to have much more responsibility than just dealing with JSON. It seems like the logic behind dealing with people should not belong in here.

Reply:
I was following IntelliJ's suggestion for the use of the diamond (<>) on line 18, but did a little bit of research and from Java 7 on one can replace type arguments for Generic Classes with
[The Diamond](https://docs.oracle.com/javase/tutorial/java/generics/types.html#diamond).

Agree that JSONDataStore has too much responsibility. I moved the logic dealing with people to a new class called People Data.